ttnn/cpp/ttnn/operations/normalization/layernorm/device/layernorm_op.cpp

```diff
@@ -131,7 +155,18 @@ void LayerNorm::validate(const std::vector<Tensor> &input_tensors, const std::ve
 }
 std::vector<tt::tt_metal::LegacyShape> LayerNorm::compute_output_shapes(const std::vector<Tensor> &input_tensors) const {
     const auto& input_tensor = input_tensors.at(0);
-    return {input_tensor.get_legacy_shape()};
+    if (this->distributed_norm_stage == DistributedLayerNormStage::PRE_ALL_GATHER)
+    {
+        auto output_shape = input_tensor.get_legacy_shape();
+        auto padding = output_shape.padding();
```

Comment:
I think we actually want 0 padding in any case, independent on the input_tensor padding (not that we support that right now, but in general)